### PR TITLE
Sprint5

### PR DESCRIPTION
**Pull request by ShipraDEVP**

### CODE REVIEW

File: .stylelintrc.yml [hunk 1]
```diff
@@ -63,27 +63,27 @@ rules:
...
-    selector-no-combinator: null
-    selector-no-id: null
+    #selector-no-attribute: null
+    #selector-no-combinator: null
+    #selector-no-id: null
     selector-no-qualifying-type: null
-    selector-no-type: null
-    selector-no-universal: true
+    #selector-no-type: null
+    #selector-no-universal: true
```
Comment (ChristianMurphy) [inline .stylelintrc.yml]: Hiding issues from the logs doesn't mean they aren't there :see_no_evil:

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 install: npm install --ignore-scripts
 script:
   - npm run lint
   # - npm test
 notifications:
   slack:
     secure: i7Do+oa/nwjb+NS3elI2RsBoePF9k9VLQB5AkiQrVxZ2OQ+q6UZ2/laH7SRpXunHFiyc3EkU52QtgtoTXER7DaWmyBoWKQDqfL2oUFM8dJJPFjVDFbu7n4XNx9AG4RE+qugREVzNwewKy49IuuQz3WF9cEjdDVhwrgRO9mVkXmC28dE8yfg8eG5tMQnHtrwTZUJVHy4ZBwZY8fOmqgD+ulVF9V0IRgACIgIm3jGePO+lSzYan+TYLWZIF1Os/RaxWUoU94Q4d5JQV63Oc5sSvJfIZVC0u7rAXp4kIR7Ru8rcuLHknIZSzYUdnTaUVaLcK6FWCELKGEMvjnWBHXFNM17VkvmZsd0Wq3EAS3igS6FxH2GgAiGKzmvIhiXM0njuLBr0IC6HFIxkz12qgWGvHI58VD2PbjkQMHS6nmDI29RNVhWCPftKWRvOGOPr4dY+hbBo+VAFliziz/B02/yUsQNqT3xqdvD3J1t4MNAEfc95X1gutpWyevakRV4FM1cOq5NMTSWCFPkzf/qygKKVK5ixUyW0H0jQhqOvMkXqZxRlKDjZQ2nClOvp/gSNm26qXJHUuuzMPFcDK2zQTBthcL5q9Ooqo1CveQat6GQdptyo3Zw5c7XI9IaE8heL9fi5gtXkubibFOaPCdt4mYPCgxJ6viDLBsD+vegDn+k0eaA=
+
+after-failure:
+  - cat /home/travis/.npm/_logs/2018-08-27T08_13_05_005Z-debug.log
```
Comment (ChristianMurphy) [inline .travis.yml]: npm logs include a timestamp, this won't work on future exceptions

File: .stylelintrc.yml (hunk 1 above)
Comment (AbhishekParmar81) [reply .stylelintrc.yml]: Hey Christian, I know commenting does not solve the issue and this pull request is not ready to be merged yet. I am looking for solutions & have found that some of the rules are not existing in our project under stylelint node module and their configurations are still in place in .stylelintrc.yml. While running the "npm run lint" travis ci was showing the error "Error: Undefined rule selector-no-attribute" for every rules which are not in our project. So meanwhile, I commented these rules to check if we are facing any other issue or not & am trying to resolve it but please suggest if you have any idea regarding the same. Below is one of the reference link which suggest to remove the unused rules from stylelint config: https://stackoverflow.com/questions/45285437/updated-stylelint-undefined-rule-selector-no-id

File: .travis.yml (hunk 2 above)
Comment (AbhishekParmar81) [reply .travis.yml]: Thank you. This was just a try to get full log from travis CI and I will change this.

File: .stylelintrc.yml (hunk 1 above)
Comment (ChristianMurphy) [reply .stylelintrc.yml]: Changelogs have a wealth of information, each of the rules was renamed.
See:
https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#780
and 
https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#7120

For a long term solution to handling upgrades, preset configs: recommended and/or standard, are a pretty good idea:
https://github.com/stylelint/stylelint-config-recommended
https://github.com/stylelint/stylelint-config-standard

Comment (AbhishekParmar81) [reply .stylelintrc.yml]: Thank you. I have applied the changes. Please check and suggest if anything is wrong.

**Comment by kgary** [comment]
Thank you @ChristianMurphy

**Pull request merged**